Mlem/App/Views/Shared/Toast/ToastView.swift

```diff
@@ -125,8 +125,8 @@ struct ToastView: View {
     @ViewBuilder
     func accountView(_ account: any Account) -> some View {
         HStack(spacing: AppConstants.doubleSpacing) {
-            AvatarView(account, showLoadingPlaceholder: false)
-                .frame(height: 27)
+            CircleCroppedImageView(account, showProgress: false)
+                .frame(width: 27, height: 27)
```

Comment:
That undesired floaty slide-in animation is back in this PR. 

iirc this was caused by using `if`/`else` statements in `FixedImageView`. When one view is swapped out for another, it doesn't retain the outgoing view's animations. I believe I fixed this using `.overlay` to control the switch between the loading placeholder and the loaded image. It might also be solvable by adding `geometryGroup` somewhere.

Reply:
Yeah, turns out it was the `CircleCroppedImageView`, not the `FixedImageView`--I've added a `.geometryGroup` to the `CircledCroppedImageView` and that seems to have fixed it.